\chapter{Introduction}

\indent The process of evolution and adaptive selection has produced cells that are capable of dynamics interactions with the environment, where information on cellular surroundings is encoded on short and long time scales. For instance, the physical association of cells with surfaces and other objects requires fast-acting cytoskeletal responses (milliseconds) to maintain cellular integrity. From this information gathered on short time scales, the cell integrates these signals through complex regulatory pathways that amplify or dampening signals from the exterior of the cell. These regulatory pathways are mediated through protein, lipid, and nucleic acid interacting factors, as the information is transduced towards decision-making 'centers' within the cell.  These 'centers' process the incoming and simultaneously begin to introduce changes in protein and nucleic acid levels, allowing the to respond successfully on long time scales (minutes to days) to the changing environment.\\
\indent As one of the most important areas that controls cellular behavior, the nucleus houses the hereditary genetic material whose expression is modulated in a highly regulated manner. By altering the rate, amount, and synchrony of gene expression across the genome, evolution has selected for gene expression changes that allow cells to respond to diverse environmental stresses in addition to intricate developmental cues. Interestingly, while the number of protein coding genes has remained fairly constant throughout metazoan evolution, the number of regulatory DNA elements has increased dramatically \cite{Levine_1710}. These and other data suggest that complex cellular choices can be mediated through fine-tuned changes in protein and RNA expression, facilitating species survival. \\
\indent Given the profound effect that changes in gene expression can have on cellular decisions, understanding the molecular basis for genome-wide regulatory decisions has served as a focal point of modern molecular biology since the discovery of the double helical nature of DNA \cite{Watson_4017}. To date, most of the proteins involved in regulating the transcription of genes into mRNA sequences have been identified through elegant experimental designs.  However, despite this information, a mechanistic understanding of these processes remains limited, and a predicative understanding of that relates protein-nucleic acid interactions to organism phenotype serves as a distant goal. In order to arrive at this deep understanding of cellular decision making, we must have a structural understanding of the proteins involved in order to appreciate the molecular determinants of cellular decision making. This thesis will pursue structural studies of a major component regulating transcription of DNA to RNA, TFIID.

\section{Metazoan gene regulation requires the coordinated assembly of protein complexes on genomic DNA}

In order for a cell to respond to specific changes to its environment, DNA sequence-specific proteins serve as the first factor responsible for altering genome-wide changes gene expression. Across the eukaryotic genome there are specific DNA sequences that are recognized by distal-acting enhancers and repressors. These regulatory regions of the genome are bound by proteins that transmit the genetically encoded DNA sequence information into changes in gene expression by affecting RNAPII activity (Figure~\ref{fig:Fig1.1}). A variety of experimental evidence has demonstrated that these sequence-specific activators and repressors are capable of ‘looping’ the genomic DNA between the enhancing DNA sequence and the promoter \cite{d_41}. Through associations between these enhancing regions of the genome and the promoter, the sequence-specific activator can directly affect the loading of RNAPII at the promoter, thus regulating gene expression.\\ 
\begin{figure}
\centering
\includegraphics[width=1\textwidth]{../Ch1_figs/Fig1.1.eps}
\caption[Model of eukaryotic transcription initiation]{Model of eukaryotic transcription initiation.  Shown here is a schematic of transcription initiation, where distal-acting enhancers (e.g. activators or repressors) bind to specific DNA sequences to activate or repress transcription activity at a specific gene.  These enhancers interact with chromatin remodelling factors (not shown here), Mediator, and TFIID to activate gene expression through long-range protein-protein contacts.  Assembling on the core promoter is the pre-initiation complex that comprises TFIID(TBP), -IIA, -IIB, -IIF, -IIE, -IIH, and RNA polymerase II.}
\label{fig:Fig1.1}
\end{figure}
\indent The initiation of transcription by RNAPII requires basal transcription factors known as TFIIA, -IIB, -IID, -IIE, -IIF, and -IIH \cite{Thomas_1201}. These factors assemble onto the core promoters of protein coding genes to form a transcription pre-initiation complex (Figure~\ref{fig:Fig1.1}) \cite{Buratowski_3748,Rhee_24}. A sequential recruitment model has been proposed whereby TFIID and Mediator serve as coactivators that facilitate interactions between upstream and promoter proximal factors \cite{Burley_3049}. The interaction of TFIID with promoter DNA can be further stabilized through a TFIIA-mediated release of the inhibitory N-terminal domain of TAF1 from the concave DNA-binding surface of TBP \cite{Bagby_2202,Geiger_2949,Liu_2574}. This facilitates the interaction of TBP with TATA box DNA, anchoring TFIID onto promoter DNA. The formation of the TFIID-TFIIA-DNA complex is then followed by the binding of TFIIB, RNAPII, TFIIF, TFIIE, and TFIIH to yield the transcriptionally competent pre-initiation complex \cite{Thomas_1201}.\\ 
\indent The transition between the process of promoter recognition and RNAPII recruitment is mediated through a TFIIB-dependent linkage of RNAPII to the TFIID-TFIIA-DNA complex. TFIIB interacts with a stirrup of TBP while also making sequence-specific DNA contacts with TFIIB-responsive elements that can be located immediately upstream and downstream of TATA boxes (Figure~\ref{fig:Fig1.5}) \cite{Lagrange_2618, Nikolov_3177}. The binding of TFIIB to the growing pre-initiation complex stabilizes TBP-DNA interactions as RNAPII is recruited through TFIIB’s extended N-terminal domain. Through interactions within RNAPII's active site and RNA exit channel \cite{Kostrewa_659}, TFIIB engages RNAPII in an inhibited state to prepare RNAPII for subsequent promoter melting steps that are mediated by TFIIE, TFIIF, and TFIIH. \\
\begin{figure}
\centering
\includegraphics[width=1\textwidth]{../Ch1_figs/Fig1.5b.eps}
\caption[TBP-templated assembly of TFIIB and TFIIA on TATA box DNA]{TBP-templated assembly of TFIIB and TFIIA on TATA box DNA. Co-crystal structures of TBP-TFIIA-DNA (PDB 1NVP) \cite{Bleichenbacher_2003} and TBP-TFIIB-DNA (PDB 1VOL) \cite{Nikolov_3177} are shown.  Side view (right) of TBP-TFIIB-DNA highlights the TBP-induced kink in TATA box DNA.}
\label{fig:Fig1.5}
\end{figure}


\section{The multi-subunit TFIID complex is essential for regulated transcription initiation}

TFIID is a multi-subunit complex that comprises TBP and about 12 to 13 additional proteins known as TAFs \cite{Burley_3049}. Initial studies with partially purified nuclear extract revealed that the 'D' nuclear extract fraction was necessary for recognition of the TATA box to support \emph{in vitro} transcription by RNAPII \cite{Matsui_3980}. After the identification and cloning TBP \cite{Buratowski_1988}, Tjian and co-workers performed a series of elegant biochemical experiments that addressed the key differences in activity between endogenously purified TFIID and recombinantly expressed TBP  (Figure~\ref{fig:Fig1.3})  \cite{Dynlacht_3551,Pugh_3586}. Utilizing reconstituted \emph{in vitro} RNAPII transcription assays, the activity of TFIID and TBP were tested on a TATA box promoter DNA template with upstream sp1 binding sites (Figure~\ref{fig:Fig1.3}). The addition of TBP allowed for the specific activation of low levels of basal transcription initiation that was further stimulated by TFIIA and unresponsive to the presence of sp1 (Figure~\ref{fig:Fig1.3}, Rxns 1 - 4). Surprisingly, unlike recombinant TBP, the endogenously purified TFIID complex was capable of responding to the presence of sp1 by stimulating higher levels of transcription initiation.  Furthermore, the addition of TFIIA to TFIID and sp1 resulted in a synergistic coactivation of sp1-stimulated transcription initiation (Figure~\ref{fig:Fig1.3}, Rxns 5 - 8). These results led to the proposal of the 'coactivator hypothesis' for TFIID \cite{Pugh_3586}, where the TAF subunits of TFIID interact with upstream activators to increase the dynamic range of the transcription output. \\
\begin{figure}
\centering
\includegraphics[width=.7\textwidth]{../Ch1_figs/Fig1.3.eps}
\caption[TFIID, not TBP, responds to upstream activators \emph{in vitro}]{TFIID, not TBP, responds to upstream activators \emph{in vitro}. Experimental results adapted from \cite{Dynlacht_3551,Pugh_3586}. Recombinantly purified TBP or immunopurified TFIID were used as promoter recognition factors for \emph{in vitro} transcription in the presence or absence of TFIIA and sp1.} 
\label{fig:Fig1.3}
\end{figure}
\indent In addition to mediating contacts with upstream activators, TFIID makes sequence-specific contacts with core promoter DNA elements through the use of TBP and TAFs. While the TATA box is the most evolutionarily conserved core promoter DNA element, sequence analysis of metazoan core promoters revealed the presence of additional elements known as Inr \cite{Smale_3697}, MTE \cite{Lim_1522}, and DPE \cite{Burke_3081} (Figure~\ref{fig:Fig1.4}). The Inr encompasses the TSS and interacts with TAF1 and TAF2 in a sequence-dependent fashion \cite{Chalkley_2339,Verrijzer_3120}. Interestingly, a reconstituted TBP-TAF1-TAF2 complex is sufficient to initiate transcription from TATA-Inr containing promoters.  Thus, these biochemical data suggest that TFIID may contain a TATA-Inr interacting module that minimally comprises TBP, TAF1, and TAF2. \\
\begin{figure}
\centering
\includegraphics[width=.7\textwidth]{../Ch1_figs/Fig1.4.eps}
\caption[Core promoter architecture]{Core promoter architecture. Promoter motifs that make sequence-specific contacts with subunits of TFIID are shown relative to the TSS (+1).  Dotted lines indicate regions of promoter DNA that interact with indicated sub-complexes of TFIID.  Note that for the downstream promoter motifs (MTE/DPE), TAF6 and TAF9 contribute to DNA binding while TAF4 and TAF12 serve as structural support (denoted *).}
\label{fig:Fig1.4}
\end{figure}
\indent Working in synergism with the Inr motif, the MTE and DPE motifs are positioned downstream of the TSS (Figure~\ref{fig:Fig1.4}). Originally identified within \emph{Drosophila}, and subsequently characterized within the human system, the MTE and DPE motifs are more commonly used across the human genome than TATA boxes \cite{Kim_1387}. Through the use of photo cross-linking studies of TFIID with promoter DNA, TAF6 and TAF9  have been identified as the subunits within TFIID that are responsible for interacting with the MTE and DPE motifs \cite{Burke_2739,Lim_1522}. While a crystal structure of the TAF6-TAF9-DNA complex remains elusive, biochemical and crystallographic data indicate that TAF6 and TAF9 heterodimerize through histone-fold domains in a manner that is homologous to histone H3 and H4 \cite{Hoffmann_2911,Xie_2805}. Interestingly, through the use of histone folds, TAF6 and TAF9 can form a higher-ordered complex that comprises TAF6/9/4/12, which is sufficient to interact with DPE-containing promoter DNA \cite{Shao_1340}. However, even though these subunits of TFIID can assemble into octameric nucleosome-like complexes, it appears that the DNA-binding residues critical for nucleosome-DNA interactions are absent within TAF6 and TAF9. This results in a TAF6/9/4/12 complex that interacts with promoter DNA through loops that extending away from the histone fold domains \cite{Shao_1340}. \\
\indent These interactions between TFIID and the core promoter elements have been exploited to create a highly active core promoter, termed the 'super core promoter' (SCP). The SCP is capable of high affinity interactions with TFIID through the presence of optimal versions of the TATA, Inr, MTE, and DPE motifs (Figure~\ref{fig:Fig1.4}) \cite{Juven-Gershon_1249}. Through the incorporation of these four promoter motifs, the SCP exhibited the highest affinity of a core promoter for human or \emph{Drosophila} TFIID to date, as measured through \emph{in vitro} transcription and DNase I footprinting \cite{Juven-Gershon_1249}. 

\section{The core promoter as a regulatory element}

The core promoter contributes to the regulatory diversity seen with metazoan genomes, playing an active role in gene regulation \cite{Juven-Gershon_468}. Initial evidence for promoter-enhancer interactions came from studies within the \emph{Drosophila} Hox gene cluster \cite{Ohtsuki_2485}. Careful comparison of enhancer and promoter architecture (e.g. presence or absence of TATA or DPE motifs) allowed the authors to identify two regulatory strategies \cite{Ohtsuki_2485}.  One strategy involved a single enhancer that indiscriminately activates genes with TATA or DPE motifs, whereas an alternative strategy appeared to be the use of a TATA-specific enhancer, where the presence of a TATA box was sufficient to confer activation by the TATA-specific enhancer (Figure~\ref{fig:Enhancers}). \\
\indent These findings have been extended through further studies in \emph{Drosophila} that identified a DPE-specific activator. Inspection of the \emph{Drosophila} Hox gene cluster revealed the presence of many DPE-containing promoters \cite{Juven-Gershon_776}. After furthe study, Caudal was identified as the transcription activator responsible for activating Hox genes in a DPE-specific manner. Additional experiments showed that Caudal specifically activated DPE- but not TATA-containing promoters, indicating that it is a DPE-specific enhancing factor. Given Caudal's role as a master regulator of body-plan development, these data suggest that core promoter DNA sequences contribute to the regulatory diversity observed across the genome (Figure~\ref{fig:Enhancers}.\\ 
\begin{figure}
\centering
\includegraphics[width=1\textwidth]{../Ch1_figs/Enhancers4.eps}
\caption[Regulation of enhancer-promoter interactions through core promoter architecture]{Regulation of enhancer-promoter interactions through core promoter architecture.}
\label{fig:Enhancers}
\end{figure}
\indent Core promoter-specific enhancers suggests that the core promoter architecture may be communicated to the enhancers through changes in the structure or activity of the GTFs. Specifically, since the TATA box and DPE motifs are recognized by subunits within TFIID, it has been proposed that changes in the architecture of core promoters may impart specific structures on TFIID \cite{Ohtsuki_2485}. Consistent with this model, RNAi knock-down of specific factors within TFIID and other transcription coactivators within \emph{Drosophila} revealed the presence of TATA and DPE activating factors within the basal transcription machinery \cite{Hsu_811}. These studies revealed that, in addition to positively regulating TATA-containing promoters, TBP could inhibit DPE-dependent transcription.  Conversely, two activators for DPE-dependent transcription - NC2 and Mot1 - inhibited transcription from TATA-containing promoters. Therefore, changes in core promoter architecture dictates specific requirements for the assembling transcription machinery on promoter DNA. These changes in machinery likely affects the activity of TFIID, since the authors identified TFIID-interacting factors (TBP, NC2 and Mot1) as key determinants of promoter selectivity. \\

\section{Electron microscopy studies of TFIID}

Despite the importance of TFIID as a coordinator of transcription initiation, high-resolution structural information has been restricted to crystal structures of a small number of subunits and domains within TFIID \cite{Bhattacharya_1103,Jacobson_2160,Kim_3416,Kim_3377,Liu_2574,Werten_1763,Xie_2805}. The size and scarcity of TFIID, typically purified from endogenous sources, has restricted structural studies to methods requiring microgram quantities of sample. Single-particle EM has proven to be an indispensable tool for the structural characterization of large multi-subunit complexes, even when sample is available in minute amounts. This technique also has the potential to characterize the structural dynamics of large protein complexes \cite{Leschziner_883}.\\
\indent EM structural studies of TFIID have yielded low-resolution structures (20 to 30 \AA) of yeast and human TFIID (Figure~\ref{fig:Compare}) \cite{Andel_2407,Brand_2375,Elmlund_691,Grob_1281,Leurent_1797,Leurent_1554,Liu_574,Papai_418,Papai_539}. A number of these studies suggested a role of conformational flexibility in promoter binding by TFIID, where large sub-domains of TFIID appear to adopt multiple conformational states. Given the low resolution of these structures, the underlying conformational flexibility of TFIID likely limits the resolution due to the need to computationally sort TFIID into a large number of classes. Recently, several groups have reported single particle EM structures of purified endogenous yeast TFIID bound to promoter DNA. These studies examined the binding of TFIID from \emph{Schizosaccharomyces pombe} and \emph{Saccharomyces cerevisiae} to promoters that contain both TATA and Inr sequence elements \cite{Elmlund_691,Papai_539}. Computational methods were used to sort TFIID into distinct conformational states, and additional densities, which were attributed to TATA box DNA, were localized to the surfaces of their structures.\\
\begin{figure}
\centering
\includegraphics[width=.7\textwidth]{../Ch1_figs/TFIID_compare.eps}
\caption[Comparison of previously obtained TFIID structures]{Comparison of previously obtained TFIID structures. Shown are 3D models for human (EMDB 1194) \cite{Grob_1281}, \emph{S. pombe} (EMDB 5134) \cite{Elmlund_691}, and \emph{S. cerevisiae} (EMDB 5176) \cite{Papai_539}. Note that the  \emph{S. cerevisiae} structure was obtained from a sample of TFIID-TFIIA-Rap1-DNA. For each 3D model, there are corresponding projections shown alongside sub-classified averages.}
\label{fig:Compare}
\end{figure}
\indent In addition to limiting the resolution of the structures, the low SNR of the images likely limits the accuracy of particle alignments. Cryo-EM analysis of human TFIID suggested the presence of multiple conformational states based upon a 3D variance-based supervised classification strategy (Figure~\ref{fig:Compare}) \cite{Grob_1281}. The resulting 2D averages and 3D models likely reflected these conformational differences, although since these data were collected at 200 kV on film under low dose conditions, there may not be a sufficient SNR to assess the validity of these structures. Importantly, given these limitations, the authors do not over interpret their structures. \\
\indent On the other hand, the yeast TFIID structures \cite{Elmlund_691,Papai_539} appear to be affected by computational errors that led the authors to over-interpret the resulting 3D models. Obtaining two structural states of \emph{S. pombe} TFIID at sub-nanometer resolution should constitute an important discovery for the field \cite{Elmlund_691}. However, given that there are not obvious alpha-helical densities within this map, the resolution limit is over-estimated. This incorrect assessment of the resolution is likely due to an over-alignment of the low SNR particles, where successive rounds of alignment with low SNR particles can impose model bias and an over-estimation of map resolution \cite{Stewart_2004}. This suggests that the conclusions drawn from the \emph{S. pombe} structure may not be based on real structural changes within \emph{S. pombe} TFIID. \\
\indent While the \emph{S. cerevisiae} structural analysis of TFIID-TFIIA-Rap1-DNA did not over-estimate the resolution, the final 3D reconstructions exhibit non-physical characteristics indicative of a computational error in the 3D alignment scheme. Throughout the structural work presented by the authors \cite{Papai_539,Papai_418}, the 3D models exhibit a hard-edge, reminiscent of an 'onion shell' (Figure~\ref{fig:Compare}). This structural feature of the 3D models is probably due to a 3D mask that has a smaller radius that the radius of the TFIID particles. Analysis of human TFIID in this incorrect fashion has resulted in similar structural features (data not shown). Imposing this incorrect mask during the 3D refinement may also have affected the particle alignments, resulting in a rotationally averaged 3D model. This is seen clearly in the 2D projections of the \emph{S. cerevisiae} DNA bound structure (Figure~\ref{fig:Compare}) where outer regions of the structure appear to be blurred. These considerations indicate that the structural conclusions proposed by the authors regarding TFIID-promoter interactions may be not accurate, due to incorrect particle alignment strategies.\\

\section{Research rationale}

In the beginning of my graduate career, the first goal of my project was to pursue a structure of human TFIID bound to promoter DNA. Given that previous work in the lab identified structural transitions existing within the human TFIID sample \cite{Grob_1281}, we hypothesized that conformational selection may be important for promoter binding. However, after a number of years of failed attempts to determine the 3D structure of TFIID-TFIIA-SCP, we were surprised to discover that the DNA binding configuration for TFIID is within a reorganized structural state. This structural reorganization was then extensively characterized using extensive 2D image analysis, in addition to obtaining an \emph{ab initio} 3D reconstruction. These studies verified that TFIID can move its lobe A (approximately 300 kDa in size) by 100\AA across its central channel in a dynamic equilibrium, providing an explanation for years of unsuccessful experiments. \\
\indent After revealing the presence of two predominant conformational states of TFIID, we discovered that the novel 'rearranged' conformation of TFIID that corresponds to a high affinity DNA binding state. By using gold labelling and multi-model refinements, we defined the organization of the rearranged conformation of TFIID bound to promoter DNA. To provide biochemical evidence for the model of DNA binding, the protein-DNA interactions were probed through DNA footprinting experiments with DNase I and MPE-Fe. By mapping the digestion patterns of wild-type and mutant SCP sequences onto the structure, we were able to show that the rearranged conformation is the DNA binding conformation for promoters of varying architecture. Our data suggest a model in which the distinct conformations of TFIID may serve as targets through which regulatory factors recruit TFIID to specific types of core promoters.\\

