%\chapter*{Appendix B: Materials \& methods}
%\addcontentsline{toc}{chapter}{Appendix B: Materials \& methods}
%\chaptermark{Appendix}
%\markboth{Appendix}{Appendix}

\chapter{Materials \& Methods}

\section{Protein preparation}
Purification of human TFIID and TFIID-TFIIA-SCP was performed as described previously \cite{Grob_1281,Liu_574}. Briefly, nuclear extract was fractionated with phosphocellulose P11 (P-Cell) resins \cite{Andel_2407,Naar_2527}.  P-Cell column fractions eluting at 1 M KCl/HEMG buffer (pH 7.9, [20 mM Hepes, 0.2 mM EDTA, 2 mM MgCl$_{2}$, 10\% glycerol] plus 1 mM DTT and 0.5 mM PMSF) were pooled, dialyzed to 0.3 M KCl/HEMG buffer (containing 0.1\% NP40 and 10 $\mu$M leupeptin) and immunoprecipitated overnight at 4$^{\circ}$C with an anti-TAF4 mAb covalently conjugated to protein G beads (GE Healthcare).  TAF4-immunoprecipitates were extensively washed with 0.65 M KCl/HEMG buffer, 0.3 M KCl/HEMG buffer and 0.1M KCl/HEMG buffer (containing 0.1\% NP40 buffer and 10 $\mu$M leupeptin) prior to the addition of human recombinant TFIIA and SCP DNA.  Purified recombinant TFIIA (see below) and SCP DNA (IDT) were incubated (2 hrs) with 2.5 $\mu$g of the washed TFIID complex bound to 250 $\mu$l of protein G sepharose beads containing covalently conjugated TAF4 mAb.  Assembled TFIID-TFIIA-SCP complexes were washed 5 times with 0.1 M KCl/HEMG buffer (containing 0.1\% NP40) and were eluted with a peptide (1 mg/ml in 0.1 M KCl/HEMG buffer/0.1\% NP40) recognized by the TAF4 mAb.  For purified TFIID used in Nanogold-labeling experiments, the final elution step included 1 mM TCEP instead of 1 mM DTT.  The eluates were concentrated with a microcon-10 concentrator, immediately applied to grids or flash frozen, and negatively stained as described below. Assembly of the TFIID-TFIIA-SCP complex involved adding a 10X-excess of recombinant TFIIA \cite{Sun_3221} and SCP DNA (IDT) prior to elution from the resin. \\
\indent Recombinant human TFIIA was purified as described previously \cite{Liu_723,Naar_2527,Sun_3221}.  TFIIA-$\alpha$/$\beta$ and TFIIA-$\gamma$ constructs were co-transformed into BL21 Star competent \emph{E. coli} (Invitrogen) and were grown until OD$_{600}$ = 0.6 â€“ 0.8.  After induction with 0.4M IPTG, the following temperature and times were used: TFIIA-$\alpha$/$\beta$ was induced at 25$^{\circ}$C for 5 hours and TFIIA-$\gamma$ was induced at 37$^{\circ}$C for 1 hour.  Cells for each construct were pelleted and washed twice in 1X PBS with 0.1 M PMSF prior to flash freezing in liquid nitrogen.  Pellets were resuspended in lysis buffer (50 mM Tris-CL pH = 7.9, 10\% glycerol, 20 mM beta-mercaptoethanol, 1 mM PMSF, 100 mM NaCl) and sonicated with 5 x 10 second prior to centrifugation to reduce viscosity.  The pellets were washed once in lysis buffer before they were extracted with lysis buffer (6 M GuHCl) for 1 hour at room temperature.  Ni$^{2+}$-NTA resin that was previously equilibrated in lysis buffer (6 M GuHCl) was added to the resuspended pellets and incubated overnight at 4$^{\circ}$C.  The resin was washed using a gravity flow column at 4$^{\circ}$C using lysis buffer (6M GuHCl) prior to elution with elution buffer (lysis buffer + 6 M GuHCl + 300 mM imidazole).  The purity of the purified and denatured TFIIA-$\alpha$/$\beta$ and TFIIA-$\gamma$ were analyzed by SDS-PAGE. TFIIA-$\alpha$/$\beta$ and TFIIA-$\gamma$ were renatured together by dialysis.  First, they were dialyzed for 4 hrs against elution buffer (HEMG: 20 mM HEPES pH = 8, 10\% glycerol, 2 mM MgCl$_{2}$, 100 mM KCl, 0.1 mM PMSF, 20 mM beta-mercaptoethanol) with the addition of 2 M GuHCl.  Using a peristaltic pump, TFIIA was renatured through the slow addition (1.3 mL/min) of elution buffer lacking GuHCl over the course of 8 hours.  Refolded TFIIA was then dialyzed against HEMG (5 mM beta-mercaptoethanol), purified over a Poros20 HQ column (Applied Biosystems), and eluted using a linear gradient of 0.1 M to 1 M KCl over 10 column volumes. \\ 
\indent For Nanogold labeling reactions, TFIIA was labeled with 1.8 nm Ni$^{2+}$-NTA-Nanogold (Nanoprobes Inc.) using proscribed protocols.  Briefly, TFIIA was incubated with a 10-fold molar excess of Ni$^{2+}$-NTA-Nanogold in a final volume of 54 $\mu$l for 20 minutes at 4$^{\circ}$C.  Prior to sample loading, a S200 size exclusion column (GE Healthcare) on an Ettan Analytical LC system was equilibrated with HEMG (1 mM TCEP). After comparison of A$_{420}$/A$_{280}$ signals, there was 50\% labeling efficiency for TFIIA. Given the labile nature of a Nanogold, Nanogold-TFIIA was used immediately for cryo-EM grid preparation.\\

\section{Nucleic acid preparation}

The DNA sequence used for the SCP was taken directly from the originally published sequence of SCP1 \cite{Juven-Gershon_1249}: GTACTTATATAAGGGGGTGGGGGCGCGTTCGTCCTCAGTCGCGATCGAACACTCGAGCCGAGCAGACGTGCCTACGGACCG. For SCP(-66), the following sequence was added immediately upstream of the SCP sequence: CTCGCGCCACCTCTGTTTTCCCAGTCACGA. Mutant promoter sequences were taken directly from previous mutational analysis of SCP1 \cite{Juven-Gershon_1249}.\\
\indent All DNA oligonucleotides were ordered through Integrated DNA Technologies (IDT) and were annealed in 10 mM Tris-HCl pH = 8.0.  For 1.4 nm-monomaleiomido-Nanogold (Nanoprobes Inc.) labeling experiments, SCP oligonucleotides were ordered with a 5'- or 3'-six carbon linked thiol functional group.  Prior to incubating DNA with Nanogold, DNA was incubated in buffer containing 1 mM TCEP for 15 minutes at room temperature to ensure that the thiol functional groups were fully reduced.  The DNA was added to a 50-fold molar excess of Nanogold that was previously solubilized in 0.2 mL distilled water and incubated for 2 hours at room temperature.  Prior to purification, the sample was concentrated using a 10 MWCO spin-column concentrator (Sartorius) from 200 $\mu$l to approximately 30 $\mu$l.  The concentrated sample was added to a S200 size exclusion column (GE Healthcare) that was previously equilibrated in running buffer (0.02 M sodium phosphate, 150 mM NaCl pH = 7.4).  After purification of a labeled DNA-Nanogold complex, comparison of A$_{420}$/A$_{260}$ signal indicated that the sample was labeled with a 70\% labeling efficiency.  Before the Nanogold-DNA sample could be added to TFIID, the sample was concentrated 5-fold and then used immediately for cryo-EM grid preparation. \\

\section{EM sample preparation} 

\indent For negative stain microscopy, four microliters of TFIID sample was directly applied to glow discharged holey carbon film covered with a continuous thin-carbon support on a 400 mesh copper grid (Electron Microscopy Sciences). For cryo-EM, samples were vitrified using a Vitrobot (Gatan, Inc.) that was set to 100\% humidity at 4$^{\circ}$C. Four microliters of sample was incubated for 30 sec. - 1 min. on carbon-thickened C-flat grid (Protochips) with 4 $\mu$m holes that were spaced 2 $\mu$m apart that had a thin-carbon support within the holes, blotted for 6.5 s, and then plunge-frozen in liquid ethane. \\
\indent Negative stain data for TFIID alone, TFIID-TFIIA, TFIID-SCP, TFIID-TFIIA-SCP, TFIID-p53, TFIID-sp1, TFIID-c-Jun, and TFIID-$\alpha$-TAF6 were collected using a Tecnai T12 bio-TWIN transmission EM operating at 120 keV under low dose (15 e$^{-}$/\AA$^{2}$) conditions on Kodak SO163 film at a nominal magnification of 30,000X from defocuses ranging from -0.70 $\mu$m to -1.20 $\mu$m. Micrographs were digitized in a Nikon Super Coolscan 8000 at a pixel size of 12.7 $\mu$m (4.23 \AA/pix). Cryo-EM data were collected on a Tecnai F20 TWIN transmission EM operating at 120 keV using a dose of 25 e$^{-}$/\AA$^{2}$ on a Gatan 4k x 4k CCD at a nominal magnification of 80,000X (1.501 \AA/pixel). Leginon software was used to automatically focus and collect exposure images \cite{Suloway_1311}.

\section{Single particle image analysis}

Negative stain data were manually picked using Boxer (EMAN) \cite{Ludtke_2307}, CTF-estimated using CTFFIND3 \cite{Mindell_1684}, and phase-flipped using SPIDER \cite{Frank_2988}. Cryo-EM data were prepared using the Appion image processing environment \cite{Lander_614} where particles were automatically picked using Signature \cite{Chen_1041}, CTF-estimated using CTFFIND3 \cite{Mindell_1684}, phase flipped using SPIDER \cite{Frank_2988}, and normalized using XMIPP \cite{Sorzano_1492}.\\
\indent For negative stain and cryo-EM data, 2D reference-free image analysis was performed using an iterative routine implementing a topology representing network \cite{Ogura_1656} in combination with mutli-reference-alignment within IMAGIC \cite{va_2849}. \\
\indent 3D refinements were performed on phase-flipped particles using an iterative projection matching and 3D reconstruction approach that was performed using libraries from EMAN2 and SPARX software packages \cite{Hohn_913,Tang_849}. 34,167 particles from cryo-EM grids prepared from TFIID-TFIIA-SCP sample were refined against low-pass filtered models of the canonical and rearranged state. For TFIID, 30,800 particles were collected and refined. For SCP(mTATA), TFIID was incubated with 10X excess of SCP(mTATA) and TFIIA, under identical conditions as the TFIID-TFIIA-SCP sample preparation, where 27,500 particles were collected and analyzed. Following this global projection matching routine, all data were further refined in FREALIGN \cite{Grigorieff_982} and low-pass filtered at the final resolution while applying a negative B-factor using \emph{bfactor.exe}. \\

\subsection{Orthogonal tilt reconstruction of TFIID-IIA-SCP} 
The substantial structural differences between the canonical and rearranged states of TFIID rendered the previously reported 3D reconstruction of the canonical state human TFIID unable to be used as a reference for projection matching and 3D reconstruction.  We thus generated an \emph{ab initio} model for the rearranged state by implementing the orthogonal tilt reconstruction (OTR) technique \cite{Leschziner_1228} on negatively stained samples of TFIID-TFIIA-SCP. \\
\indent Tilt pairs at +/- 45$^{\circ}$ were collected on a TFIID-IIA-SCP sample prepared on glow discharged holey carbon film \cite{Bradley_4010} on a 400 mesh copper grid (Electron Microscopy Sciences) with a thin continuous carbon layer using a Tecnai F20 TWIN transmission electron microscope operating at 120 keV using a dose of 25 e$^{-}$/\AA$^{2}$ on a Gatan 4k x 4k CCD at a nominal magnification of 80,000X (1.501 \AA/pixel).  Particles were manually picked from the tilt pairs using \emph{xmipp mark} within XMIPP \cite{Sorzano_1492}.  The XMIPP coordinates (.pos) were converted to BOXER coordinates (.box) prior to particle extraction using â€˜batchboxerâ€™ within EMAN \cite{Ludtke_2307}.  The tilt angles for each micrograph were calculated using CTFTILT \cite{Mindell_1684}.  Extracted particles from a given tilt were subjected to 2D reference-free classification and alignment using multivariate-statistical-analysis and multi-reference-refinement within IMAGIC \cite{va_2849}.  To calculate individual class volumes, euler angles were calculated using the equations previously described \cite{Leschziner_452}, 3D reconstructions were performed using 'bp 3f' within SPIDER \cite{Frank_2988}, and the volumes were filtered based upon FSC = 0.5.\\   
\indent Since OTR preserves three-dimensional features of individual class volumes without suffering from the 'missing-wedge' problem of random conical tilt \cite{Chandramouli_280}, individual class volumes were refined against untilted negative stain data for TFIID-IIA-SCP.  These refined models were then quantitatively compared to the characteristic class averages of TFIID in six characteristic conformations and models were excluded based upon Fourier Ring Correlation (FRC) \cite{Saxton_3960} values below 60 \AA.  A majority (8/10) of the refined, validated models were found to be in the rearranged state, whereas a minority (2/10) were in the canonical conformation.\\ 

\subsection{Focused classification of lobe A}
The overall data processing strategy for assessing lobe A's conformational heterogeneity is mapped out in Figure \ref{fig:Fig2.5}.  Briefly, negatively stained particles for each dataset - TFIID (16,000 particles), TFIID-SCP (9,859 particles), TFIID-IIA-SCP (19,678 particles), and TFIID-IIA (20,152 particles) â€“ were aligned and classified in a reference-free fashion using a topology representing network \cite{Ogura_1656} and multi-reference-alignment within IMAGIC \cite{va_2849}, where class average reference images were aligned to each other and mirrors generated after each iteration using the â€˜prep-mra-refsâ€™ command within IMAGIC.  After 8-9 iterations, all particles corresponding to the characteristic view of TFIID were extracted, mirrored if necessary, and placed into a single stack of particles.  These particles were then aligned to a masked reference of the BC core using multi-reference-alignment within IMAGIC \cite{va_2849}.  Multivariate-statistical-analysis was performed within a mask drawn around lobe A.  The resulting eigen-images reflected the conformational flexibility of lobe A and were used to perform hierarchical ascendant classification.  Subsequently generated classums (10-20 particles/class) showed a range of positions for lobe A relative to the rigid BC core.\\
\indent To measure lobe A's position, individual classums were measured using BOXER within EMAN \cite{Ludtke_2307} and plotted within MATLAB. Boxes were placed, in order, on lobes B, C, and A.  The coordinates were used to calculate a projection of lobe A's position on the B-C axis that were normalized to the length of the BC core.  A histogram of lobe A positions were calculated within MATLAB and the distribution was fitted using a maximum-likelihood estimate of a mixed Gaussian distribution ('gmdistribution.fit') and was plotted using 'pdf.' Wilcoxon rank sum tests were performed within MATLAB ('ranksum') to test for statistical differences between distributions of lobe A positions from the different datasets.\\

\subsection{Cryo-EM of Nanogold complexes \& image analysis}
 
For localization of Nanogold labels within cryo-EM images of TFIID-IIA(gold)-SCP (10,075 particles), TFIID-IIA-SCP(Nanogold at TATA) (851 particles), and TFIID-IIA-SCP(Nanogold at +45) (6,502 particles), focal pair images were collected using Leginon \cite{Suloway_1311} at defocuses of -3 $\mu$m and -0.5$\mu$m on a Tecnai F20 TWIN transmission electron microscope operating at 120 keV using a dose of 25 e$^{-}$/\AA$^{2}$ on a Gatan 4k x 4k CCD at a nominal magnification of 80,000X (1.501 \AA/pixel).  Particles were manually picked using BOXER, shifts between focal pairs were calculated using â€˜alignhugeâ€™ and applied to the particle coordinate files prior to particle extraction with â€˜batchboxerâ€™ within EMAN \cite{Ludtke_2307}.  The high defocus particles were then normalized and dusted using \emph{xmipp normalize} within XMIPP \cite{Sorzano_1492} to remove gold signal $>$ 3.5$\sigma$. These dusted, high defocus particles were subjected to reference-free 2D classification using iterative classification and alignment within IMAGIC with multivariate statistical analysis and multi-reference-alignment \cite{va_2849}. The rotation and shifts applied to the high-defocus particles were applied to the following stacks of particles using 'equivalent-rotation' within IMAGIC:  1) un-dusted high defocus particles termed 'high defocus particles' that were normalized using 'edgenorm' within the \emph{proc2d} command of EMAN; 2) thresholded high defocus particles where pixels $>$ 4$\sigma$ were converted to 1 and all pixels below 4$\sigma$ were converted to 0 for each un-dusted high defocus particle; 3) un-dusted low defocus particles termed 'low defocus particles' that were normalized using 'edgenorm' within the \emph{proc2d} command of EMAN; and 4) thresholded low defocus particles where pixels $>$ 4$\sigma$ were converted to 1 and all pixels below 4$\sigma$ were converted to 0 for each un-dusted low defocus particle.  By averaging low defocus particles and low defocus thresholded particles according to the 2D reference-free alignment of dusted high defocus particles, Nanogold density could be localized with high confidence given the high contrast of Nanogold at low defocus values.

\section{DNase I and MPE-Fe footprinting}

Extended-length M13 sequencing primers (28 nt) with 5'-ends corresponding to basepair -109 and +110 (relative to the start site of SCP as +1) were purified by denaturing gel electrophoresis, 5'-end-labeled with [$\gamma$-$^{32}$P]-ATP and T4 polynucleotide kinase, desalted on a 900 $\mu$l Sephadex G50 fine column, extracted with phenol/CHCl$_{3}$/isoamyl alcohol (IAA), precipitated with ethanol, and the pellet was suspended in 10 mM Tris-Cl pH 8.0, 0.1 mM EDTA. DNA probes with either a 5'-upstream or a 5'-downstream labeled end were generated by PCR using the appropriate combination of labeled and unlabeled primers with pUC119-SCP1 \cite{Juven-Gershon_1249} as template.  The 220 basepair product was isolated by native gel electrophoresis, passively eluted into 10 mM Tris-Cl pH 8.0, 0.1 mM EDTA, 0.2\% SDS, 1M LiCl  followed by phenol/CHCl$_{3}$/IAA extraction, ethanol precipitation, and resuspension in 10 mM Tris-Cl pH 8.0, 0.1 mM EDTA. A+G, T, and A chemical sequencing ladders used for mapping cleavage sites followed published methods \cite{Iverson_3815,Sambrook_3729}.\\
\indent Protein-DNA complexes were formed for 20 min. at 30$^{\circ}$C in 20 $\mu$l of binding buffer containing 20 mM KHEPES pH 7.8, 4 mM MgCl$_{2}$, 0.2 mM EDTA, 0.05\% (v/v) NP-40, 8\% (v/v) glycerol, 100 $\mu$g/mL BSA, and 1 mM (DNase I footprinting) or 2 mM DTT (MPE-Fe footprinting) upon final assembly containing 2 nM TFIID, 20 nM TFIIA, and 0.2 nM (DNase I footprinting) or 0.75 nM (MPE-Fe footprinting) DNA probe.  Proteins were diluted in a diluent consisting of binding buffer with 20\% glycerol and 200 $\mu$g/mL BSA.  DNase I digestion was initiated by the addition of 2.2 $\mu$l of 0.38 - 0.75 mU/$\mu$l DNase I (in diluent containing 5 mM CaCl$_{2}$) and terminated 30 seconds later by the addition of 158 $\mu$l of a stop solution (10 mM Tris-Cl pH 8.0, 3 mM EDTA, 0.2\% SDS).  MPE-Fe(II) was generated by combining equal volumes of 1 mM methidiumpropyl EDTA (Sigma, discontinued) and 1 mM NH$_{4}$Fe(II)SO$_{4}$ (Aldrich) for 5 min. followed by dilution in water.  MPE-Fe(II) cleavage was initiated by the sequential addition of 1.2 $\mu$l 23 mM NaAscorbate and 1.2 $\mu$l 46 $\mu$M MPE-Fe(II) and terminated 2 min. later by the sequential addition of 2 $\mu$l 120 $\mu$M bathophenanthroline and 156 $\mu$l stop solution.  Samples were processed by phenol/CHCl$_{3}$/IAA extraction and ethanol precipitation.  Digestion products were resolved on 10\% polyacrylamide (37.5:1) containing 8.3 M urea until xylene cyanol migrated 80\% down the gel.  The phosphor image (Typhoon Trio; GE Healthcare) was analyzed using Image Gage (Fuji Film). 


